ci-operator/step-registry/jboss-fuse/run-tests/jboss-fuse-run-tests-commands.sh

```diff
@@ -0,0 +1,198 @@
+              subPath: maven-repo
+EOF
+
+sleep 30
+done
+}
+
+function check_tests() {
+	mapfile -t podlist < <(oc get pods -n jboss-fuse-interop -l application=xpaas-qe --no-headers=true | awk '{print $1}')
+	echo "LIST ----"
+	echo "${podlist[@]}"
+
+	  while (( ${#podlist[@]} )); do
+	    echo "Number running pods still: ${#podlist[@]}"
+	    for i in "${!podlist[@]}"; do
+	       echo "Evaluating ${podlist[$i]} ..."
+	       podlog=$(while read line; do echo $line; done <<< "$(oc logs --tail=50 ${podlist[$i]} -n jboss-fuse-interop)")
+	       if [[ "$podlog" == *"BUILD SUCCESS"* ]]; then
+	         echo "pod ${podlist[$i]} terminated"
+           if [[ "$podlog" == *"Failures: 0, Errors: 0"* ]]; then
+             echo "${podlist[$i]} was successful"
+             todelete=$(echo ${podlist[$i]} |  awk -F"[-]" '{print $1}')
+             echo "DC TO DELETE: ${todelete} WAIT 10 seconds"
+             sleep 10
+             oc delete dc/${todelete} -n jboss-fuse-interop
+           else
+             echo "${podlist[$i]} was unsuccessful at first attempt, rerun #1"
+             oc -n jboss-fuse-interop exec ${podlist[$i]} -- rm -rf /tmp/log/${podlist[$i]}
+             oc -n jboss-fuse-interop exec ${podlist[$i]} -- rm -rf /tmp/surefire-reports/${podlist[$i]}
+             oc delete pod ${podlist[$i]} -n jboss-fuse-interop
+           fi
+         fi
+	       if [[ "$podlog" == *"BUILD FAILURE"* ]]; then
+	       	 echo "${podlist[$i]} was in build failure at first attempt, rerun #1"
+	       	 oc -n jboss-fuse-interop exec ${podlist[$i]} -- rm -rf /tmp/log/${podlist[$i]}
+           oc -n jboss-fuse-interop exec ${podlist[$i]} -- rm -rf /tmp/surefire-reports/${podlist[$i]}
+	       	 oc delete pod ${podlist[$i]} -n jboss-fuse-interop
+	       fi
+	    done
+	    sleep 120
+		  mapfile -t podlist < <(oc get pods -n jboss-fuse-interop -l application=xpaas-qe --no-headers=true | awk '{print $1}')
+	  done
+}
+
+function copy_logs() {
+  oc rollout latest dc/nginx-server -n jboss-fuse-interop
+  sleep 120
+  NGINX_POD=$(oc get pods -n jboss-fuse-interop -l application=nginx --no-headers=true | awk '{print $1}')
+  echo "get logs from pod ${NGINX_POD}"
+  oc -n jboss-fuse-interop exec ${NGINX_POD} -c nginx -- mkdir -p /tmp/reports
+  oc -n jboss-fuse-interop exec ${NGINX_POD} -c nginx -- find /tmp/surefire-reports/ -name *.xml -exec cp {} /tmp/reports \;
+  KUBEADMIN_PWD=$(cat $SHARED_DIR/kubeadmin-password)
+  oc -n jboss-fuse-interop exec ${NGINX_POD} -c nginx -- find /tmp/reports/ -name *.xml -exec sed -i "s/\"$KUBEADMIN_PWD\"/\"XXXXXXXX\"/g" {} \;
+  echo "surefire logs stored"
+  mkdir -p ${ARTIFACT_DIR}/tests
+  echo "creating test log sub-folder"
+  oc cp jboss-fuse-interop/"${NGINX_POD}":/tmp/log "${ARTIFACT_DIR}/tests"
+  oc cp jboss-fuse-interop/"${NGINX_POD}":/tmp/reports "${ARTIFACT_DIR}"
+  oc -n jboss-fuse-interop exec ${NGINX_POD} -c nginx -- cd ${ARTIFACT_DIR}; for filename in *.xml; do mv "$filename" "junit_$filename"; done
```

Comment:
@smongiar the logs and xmls are already copied to ARTIFACT_DIR from Lines 185-186 (see [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/38586/rehearse-38586-periodic-ci-jboss-fuse-fuse-xpaas-qe-container-main-jboss-fuse-ocp4.14-lp-interop-jboss-fuse-interop-aws/1696496773628956672/artifacts/jboss-fuse-interop-aws/jboss-fuse-run-tests/artifacts/)) so in my suggestion this should work for renaming xmls to `junit_*.xml`
```suggestion
  rename TESTS junit_TESTS ${ARTIFACT_DIR}/TESTS*.xml
  ```

Reply:
Hi @chetna14manku last time I tried with the `rename` command, but I got `Command 'rename' not found`... I can give it a try again and let see together